akalisten/templates/lotsude/style.css

```diff
@@ -91,6 +91,13 @@ footer {
     justify-content: space-between;
 }
 
+.mucken-header {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    gap: 10px;
+}
```

Comment:
I don't see why it would add value if the button for entering in a mucke would take space away from the unfolded info box in tablet and desktop view. Folded up and in mobile view, however, it looks good.
Perhaps the entry link would be better placed in accordion-header next to the title if you would like to have it on one level.
Alternatively, I would really just leave it in its original position and enlarge it to full width in the mobile view.

Reply:
I think it just looked a bit weird to me to have the short button right above the very wide accordion header. But I get your point. I'll open a new PR that just adds the styling without any layout changes